src/Preferences.cpp

```diff
@@ -8,6 +8,7 @@
 
 BLocker prefsLock;
 BMessage gPreferences;
+rgb_color gNegColor;
```

Comment:
```suggestion
rgb_color gNegativeColor;
```

Reply:
Thanks for the review. PR updated.

@dospuntos: I never know if you're notified of new PRs or only become aware of them when I mention your nick... You're always invited to review, of course. :)

Reply:
@humdingerb thanks for the heads up, I do receive notifications.

BTW, I'm trying to implement currency localization using the Haiku locale settings, and that means changes in a lot of files. Just letting you know in case you're planning on making more changes, in which case we should try to avoid too many merge conflicts...

Reply:
I think I'll only have small changes from playing around a bit more. You go ahead, I'll rebase my changes after yours are merged.